app/assets/scripts/components/common/layers/layer-co2.js

```diff
@@ -31,19 +31,20 @@ export default {
     source: {
       type: 'raster',
       tiles: [
-      `${config.api}/{z}/{x}/{y}@1x?url=s3://covid-eo-data/xco2/xco2_15day_base.{date}.tif&resampling_method=bilinear&bidx=1&rescale=0.0004%2C0.00042&color_map=reds&color_formula=gamma r {gamma}`
+      `${config.api}/{z}/{x}/{y}@1x?url=s3://covid-eo-data/xco2/xco2_15day_base.{date}.tif&resampling_method=bilinear&bidx=1&rescale=0.000408%2C0.000419&color_map=rdylbu_r&color_formula=gamma r {gamma}`
       ]
     }
   },
 
   swatch: indicatorGroupColors['greenhouse-gas'],
   legend: {
     type: 'gradient-adjustable',
-    min: 'less',
-    max: 'more',
+    min: '408',
+    max: '419',
     stops: [
-      '#FFFFFF',
-      '#FF0000'
+      '#FF0000',
+      '#FFFF00',
+      '#0000FF'
```

Comment:
Current colors have too much contrast.
![image](https://user-images.githubusercontent.com/1090606/85314279-2ddeab00-b4b1-11ea-8a29-e3b75d527272.png)

These seem to be more appropriate:
```suggestion
      '#5D4FA2',
      '#2F75BE',
      '#6DC7A3',
      '#D7ED96',
      '#FFEA9B',
      '#FA894C',
      '#B11E4D'
```
![image](https://user-images.githubusercontent.com/1090606/85314147-f3750e00-b4b0-11ea-9c81-15e9ca576899.png)